package.json

```diff
@@ -1890,6 +1891,7 @@
         "esbuild-loader": "2.13.1",
         "eslint": "^7.28.0",
         "eslint-config-prettier": "8.3",
+        "eslint-plugin-aws": "file:plugins/eslint-aws",
```

Comment:
let's mention this in https://github.com/aws/aws-toolkit-vscode/blob/master/CONTRIBUTING.md#tooling

```
### AWS eslint plugin

Defines custom lint rules for the project.
```